eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/AbstractEventsourcedView.scala

```diff
@@ -151,6 +153,20 @@ abstract class AbstractEventsourcedView(val id: String, val eventLog: ActorRef)
       "Use commandContext.become(...) to change it.")
 
```

Comment:
To align with the new Akka-Java API of `AbstractActor` I would suggest to add the following methods:
```
def getContext(): AbstractActor.ActorContext
def getSelf(): ActorRef
def getSender(): ActorRef
```